src/apiClient.ts

```diff
@@ -106,9 +106,7 @@ export default class APIClient {
           }
           url.searchParams.set('metadata_pair', metadataPair.join(','));
         } else if (Array.isArray(value)) {
-          for (const item of value) {
-            url.searchParams.append(snakeCaseKey, item as string);
-          }
+          url.searchParams.append(snakeCaseKey, value.join(','));
```

Comment:
@wallacecs007 can you write a test? Also, will this apply only to the `anyEmail` query param or all?

Reply:
@tjovicic I fixed the test that was failing due to this change and this will apply to all query parameters. I don't see any case in our API spec docs where we would want multiple of the same query parameter passed in to Nylas outside of Metadata.

Metadata has it's own case and will be unaffected.